crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -144,11 +144,34 @@ impl SweetConductor {
         keystore: Option<MetaLairClient>,
         rendezvous: Option<R>,
     ) -> SweetConductor
+    where
+        C: Into<SweetConductorConfig>,
+        R: Into<DynSweetRendezvous> + Clone,
+    {
+        Self::create_with_defaults_and_metrics(config, keystore, rendezvous, false).await
+    }
+
+    /// Create a SweetConductor with a new set of TestEnvs from the given config
+    /// and a metrics initialization.
+    pub async fn create_with_defaults_and_metrics<C, R>(
+        config: C,
+        keystore: Option<MetaLairClient>,
+        rendezvous: Option<R>,
+        with_metrics: bool,
+    ) -> SweetConductor
     where
         C: Into<SweetConductorConfig>,
         R: Into<DynSweetRendezvous> + Clone,
     {
         let rendezvous = rendezvous.map(|r| r.into());
+        let dir = TestDir::new(test_db_dir());
+
+        if with_metrics {
+            let mut path = std::path::PathBuf::from(dir.as_ref());
+            path.push("influxive");
```

Comment:
Similarly here, I would rather not add to the API surface of `sweettest` and control enabling metrics with an env var instead

Reply:
Again that's similar to how we init tracing at the top of a lot of tests but it doesn't do anything unless `RUST_LOG` is set

Reply:
I can add an environment variable to disable metrics... but I'd really like them to be always on, so people can forward their metrics databases in bug reports when they encounter bugs in launcher.

Reply:
I'll rephrase 'enable' because that's not as specific as I should have been. I meant a env var to pick which metrics back-end to enable. I understand that the aim is to have this always on unless disabled so maybe influx can be an option and the default, but an environment variable can enable other methods too. Or an enum, or anything else that let's this method not take a path as part of its interface because prometheus/stdout/zpages etc don't require a path

Reply:
Okay, I removed the config for disabling metrics and switched it to an environment variable that does it. Obviously I can't create any environment variables for enabling other metrics collector types until we have them, so we can figure that out whenever we're ready to do that.

If influx is to be the default it needs to take the path... other things that don't need it can just ignore it. It wouldn't be much of a default if you had to specify an environment variable to make it work. Not to mention, that is impossible since sometimes the env dir is a temp-dir that only holochain knows about, so there'd be no way for a user to specify it in the environment.